tests/acceptance/features/bootstrap/PublicWebDavContext.php

```diff
@@ -332,17 +332,12 @@ public function downloadPublicFileInsideAFolderWithRange(string $path, string $r
 	/**
 	 * @param string $path
 	 * @param string $user
-	 * @param bool $shareNg
 	 *
 	 * @return ResponseInterface
 	 */
-	public function downloadFromPublicLinkAsUser(string $path, string $user, bool $shareNg = false): ResponseInterface {
+	public function downloadFromPublicLinkAsUser(string $path, string $user): ResponseInterface {
```

Comment:
this method might be used somewhere with $shareNg arg
```suggestion
	public function downloadFromPublicLinkAsUser(string $path, string $user): ResponseInterface {
```

Reply:
The method is used only once without passing `$shareNg` so it's safe to remove. Also, the condition in L#340 is changed.